### PR TITLE
feat: MON-13253 standardization of invalid license messages with SVG as assets

### PR DESCRIPTION
**Pull request by JKancel**
## Description

This PR introduced a new background template svg component to style error pages (it's a rework of this PR : https://github.com/centreon/centreon-frontend/pull/204, svg are now based on assets loaded by `url-loader`) :

https://user-images.githubusercontent.com/6154460/189327166-bb6d329a-3957-49a0-acc2-5d4708ef8ba4.mov

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Go to `centreon-ui` folder,
- Launch storybook with the following command : ``` npm run storybook ```
- Navigate to [navigate to your local storybook instance](http://localhost:9001/?path=/story/fallbackpage-licenceinvalid--licence-invalid)
- Try the fallback page by editing its props.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: packages/centreon-ui/.storybook/preview-body.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<style>
+  html, body {
+    height: 100vh;
+  }
+  #root {
+    height: 100%;
+  }
+</style>
```
Comment (Thebarda) [inline packages/centreon-ui/.storybook/preview-body.html:8]: BTW, you can use the CssBaseline component from Material UI

**Comment by bdauria** [comment]
@JKancel you have some Component files that aren't named using PascalCase

File: packages/centreon-ui/src/Logo/centreonLogo.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { makeStyles } from 'tss-react/mui';
+
+import { CentreonLogo } from './centreonLogo';
+
+const useStyles = makeStyles()(() => ({
+  logo: {
+    width: '350px',
```
Comment (Thebarda) [inline packages/centreon-ui/src/Logo/centreonLogo.stories.tsx:8]: The width is fixed maybe it's better to make it responsive

File: packages/centreon-ui/src/Logo/centreonLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <path d="M358.1 37.3c.4-6.4 4.8-10.7 11.3-10.7 6.6 0 10.3 4.4 10.5 10.7h-21.8zm32.6 7.9v-3.8c0-12.9-7-23.5-21.2-23.5-13.2-.1-22.5 10.1-22.5 24 0 1.1.1 2.2.2 3.3 1.3 12.1 10.1 20.6 23.2 20.6h.1c8.5 0 14.5-2.8 19-8.3l-7.6-6.2c-2.1 2.9-6 5.3-11.4 5.3-7.7 0-12.1-4-12.4-11.4h32.6z" />
+        <path d="M486.7 64.9h11.1V38.7c0-11.5-9.3-20.9-20.8-20.9-4.5 0-8.9 1.4-12.5 4.1-2-2.6-5.2-4.1-8.4-4.1v47.1h11V38.6c0-5.4 4.5-9.8 9.9-9.8s9.7 4.4 9.8 9.7l-.1 26.4z" />
+        <path d="M446.8 39.2c0-.2-.1-.4-.1-.6-.1-.6-.2-1.3-.3-1.9 0-.2-.1-.3-.1-.5-.2-.6-.3-1.3-.5-1.9 0-.1-.1-.2-.1-.3-.2-.7-.5-1.3-.7-2 0-.1 0-.1-.1-.1-.3-.7-.6-1.4-1-2-4.3-7.6-12.3-12.4-21.1-12.4s-16.8 4.7-21.1 12.4c-.4.7-.7 1.3-1 2l-.1.1c-.3.6-.5 1.3-.7 2 0 .1-.1.2-.1.3-.2.6-.4 1.3-.5 1.9 0 .1-.1.3-.1.5-.1.6-.2 1.3-.3 1.9 0 .2-.1.4-.1.6-.2 1.7-.2 3.3 0 5 .2 1.6.5 3.2 1 4.7.2.8.5 1.5.8 2.2 3.8 8.9 12.6 14.8 22.3 14.7 13.3 0 24.1-10.8 24.1-24.1v-.1c0-.8-.1-1.6-.2-2.4m-24 17.2c-8.1 0-14.8-6.6-14.8-14.8 0-8.1 6.6-14.8 14.8-14.8 8.1 0 14.8 6.6 14.8 14.8-.1 8.2-6.7 14.8-14.8 14.8" />
+      </g>
+    </svg>
```
Comment (Thebarda) [inline packages/centreon-ui/src/Logo/centreonLogo.tsx:38]: Load it as an image

File: packages/centreon-ui/src/Module/LicensedModule/LicenseCheck/index.tsx [hunk 4]
```diff
@@ -3,8 +3,7 @@ import * as React from 'react';
 import { isNil } from 'ramda';
 import { useTranslation } from 'react-i18next';
 
-import { Alert, AlertTitle, Container } from '@mui/material';
-
+import { FallbackPage } from '../../../FallbackPage/fallbackPage';
```
Comment (Thebarda) [inline packages/centreon-ui/src/Module/LicensedModule/LicenseCheck/index.tsx:6]:
```suggestion
import { FallbackPage } from '../../../FallbackPage/FallbackPage';
```

**Review by Thebarda — COMMENTED** [review]
You can remove the junit.xml

**Comment by Thebarda** [comment]
> > @JKancel you have some Component files that aren't named using PascalCase
> 
> @bdauria `PascalCase` is in your conventions for folders name and filesname too ? I find a lot of `index.ts` so I assumed that filename was in `camelCase`.

I think we must stop using `index.tsx` and rename them using the same name as the folder. It would make it clearer to read/find/use the good convention to create component

**Comment by JKancel** [comment]
> > > @JKancel you have some Component files that aren't named using PascalCase
> > 
> > @bdauria `PascalCase` is in your conventions for folders name and filesname too ? I find a lot of `index.ts` so I assumed that filename was in `camelCase`.
> 
> I think we must stop using `index.tsx` and rename them using the same name as the folder. It would make it clearer to read/find/use the good convention to create component

Totally agree. But we should discuss about the naming convention, using PascalCase for file name just because there is a component inside which is declared following PascalCase imho is not the best practice. I found some articles about this case : https://www.robinwieruch.de/react-folder-structure/

**Comment by bdauria** [comment]
> > > > @JKancel you have some Component files that aren't named using PascalCase
> > > 
> > > 
> > > @bdauria `PascalCase` is in your conventions for folders name and filesname too ? I find a lot of `index.ts` so I assumed that filename was in `camelCase`.
> > 
> > 
> > I think we must stop using `index.tsx` and rename them using the same name as the folder. It would make it clearer to read/find/use the good convention to create component
> 
> Totally agree. But we should discuss about the naming convention, using PascalCase for file name just because there is a component inside which is declared following PascalCase imho is not the best practice. I found some articles about this case : https://www.robinwieruch.de/react-folder-structure/

To be honest, I'm more on the side of naming Components folder using Pascal case so that it can be identified instantly as such within the codebase (in comparison to utils or other non Component elements). But what the article says about the way it's handled by some OS is painfully true, and we had some trouble in the past while renaming folders (from camel to Pascal) which was handled correctly on macOS but not on Linux.

I'm open to discuss it later on though. How about we stick to the current convention for now and have a chat the three of us later on to state about this? 
Be careful now with renaming the directories though, you might want to to it directly in your filesystem ^^

**Review by Wedierre — APPROVED** [review]

**Comment by JKancel** [comment]
After several ci builds, I notice that there are size differences between storyshot produced in my local machine compared to those from Jenkins (os is certainly the cause). I add some config to fix the viewport to 1000x1000 for the moment, but I found a better approach based on docker : https://github.com/spring-media/storybook-addon-image-snapshots#testing-with-a-local-running-storybook-server
Maybe we could explore it in a futur iteration.

**Review by Thebarda — APPROVED** [review]

**Review by Wedierre — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**